tests/e2e/cucumber/features/smoke/spaces/publicLink.feature

```diff
@@ -87,9 +87,15 @@ Feature: spaces public link
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" closes the public link password dialog box
     When "Alice" tries to sets the password of the public link named "myPublicLink" of resource "lorem.txt" to "12345678"
     Then "Alice" should see an error message
       """
       Unfortunately, your password is commonly used. please pick a harder-to-guess password for your safety
       """
+    And "Alice" shows the password of the public link
+    And "Alice" hides the password of the public link
+    And "Alice" copies the filled password
+    And "Alice" generates the password for the public link
+    And "Alice" closes the public link password dialog box
```

Comment:
instead of closing dialog, please set generated password so that we have a complete case

I would suggest this sequence of user actions: 
- `And "Alice" generates the password for the public link`
- `And "Alice" copies the filled password`
- and press button `set` 

<img width="506" alt="Screenshot 2023-10-19 at 12 27 24" src="https://github.com/owncloud/web/assets/84779829/5515ac95-a6bf-4b75-91cd-ce1714ef7922">

You can also check the PUT response that the link with the generated password has been created. 
As an additional check-> you can check that the copy buffer is equal to the request body(password) - it's optional